wled00/ir.cpp

```diff
@@ -70,10 +70,9 @@ void decBrightness()
   }
 }
 
-// apply preset or fallback to a effect and palette if it doesn't exist
 void presetFallback(uint8_t presetID, uint8_t effectID, uint8_t paletteID)
```

Comment:
As this is now just a duplicate of `applyPresetWithFallback()`, we should remove it